CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -1739,57 +1720,49 @@ bool SiPixelTemplate::interpolate(int id, float cotalpha, float cotbeta, float l
+              yxratio * ((1.f - xxratio) * entx20_->pixmax + xxratio * entx22_->pixmax);
 
-    pixmax_ = (1.f - yxratio) * ((1.f - xxratio) * thePixelTemp_[index_id_].entx[iylow][ilow].pixmax +
-                                 xxratio * thePixelTemp_[index_id_].entx[iylow][ihigh].pixmax) +
-              yxratio * ((1.f - xxratio) * thePixelTemp_[index_id_].entx[iyhigh][ilow].pixmax +
-                         xxratio * thePixelTemp_[index_id_].entx[iyhigh][ihigh].pixmax);
-
-    r_qMeas_qTrue_ = (1.f - yxratio) * ((1.f - xxratio) * thePixelTemp_[index_id_].entx[iylow][ilow].r_qMeas_qTrue +
-                                        xxratio * thePixelTemp_[index_id_].entx[iylow][ihigh].r_qMeas_qTrue) +
-                     yxratio * ((1.f - xxratio) * thePixelTemp_[index_id_].entx[iyhigh][ilow].r_qMeas_qTrue +
-                                xxratio * thePixelTemp_[index_id_].entx[iyhigh][ihigh].r_qMeas_qTrue);
+    r_qMeas_qTrue_ = (1.f - yxratio) * ((1.f - xxratio) * entx00_->r_qMeas_qTrue + xxratio * entx02_->r_qMeas_qTrue) +
+                     yxratio * ((1.f - xxratio) * entx20_->r_qMeas_qTrue + xxratio * entx22_->r_qMeas_qTrue);
 
     for (i = 0; i < 4; ++i) {
-      xavg_[i] = (1.f - yxratio) * ((1.f - xxratio) * thePixelTemp_[index_id_].entx[iylow][ilow].xavg[i] +
-                                    xxratio * thePixelTemp_[index_id_].entx[iylow][ihigh].xavg[i]) +
-                 yxratio * ((1.f - xxratio) * thePixelTemp_[index_id_].entx[iyhigh][ilow].xavg[i] +
-                            xxratio * thePixelTemp_[index_id_].entx[iyhigh][ihigh].xavg[i]);
-      if (flip_x) {
+      xavg_[i] = (1.f - yxratio) * ((1.f - xxratio) * entx00_->xavg[i] + xxratio * entx02_->xavg[i]) +
+                 yxratio * ((1.f - xxratio) * entx20_->xavg[i] + xxratio * entx22_->xavg[i]);
+      if (flip_x_) {
         xavg_[i] = -xavg_[i];
       }
 
-      xrms_[i] = (1.f - yxratio) * ((1.f - xxratio) * thePixelTemp_[index_id_].entx[iylow][ilow].xrms[i] +
-                                    xxratio * thePixelTemp_[index_id_].entx[iylow][ihigh].xrms[i]) +
-                 yxratio * ((1.f - xxratio) * thePixelTemp_[index_id_].entx[iyhigh][ilow].xrms[i] +
-                            xxratio * thePixelTemp_[index_id_].entx[iyhigh][ihigh].xrms[i]);
+      xrms_[i] = (1.f - yxratio) * ((1.f - xxratio) * entx00_->xrms[i] + xxratio * entx02_->xrms[i]) +
+                 yxratio * ((1.f - xxratio) * entx20_->xrms[i] + xxratio * entx22_->xrms[i]);
 
-      //	      xgx0_[i]=(1.f - yxratio)*((1.f - xxratio)*thePixelTemp_[index_id_].entx[iylow][ilow].xgx0[i] + xxratio*thePixelTemp_[index_id_].entx[iylow][ihigh].xgx0[i])
-      //		          +yxratio*((1.f - xxratio)*thePixelTemp_[index_id_].entx[iyhigh][ilow].xgx0[i] + xxratio*thePixelTemp_[index_id_].entx[iyhigh][ihigh].xgx0[i]);
+      //	      xgx0_[i]=(1.f - yxratio)*((1.f - xxratio)*entx00_->xgx0[i] + xxratio*entx02_->xgx0[i])
```

Comment:
Is this commented out code needed: add one line explaining its usage, if so; otherwise just remove

Reply:
removed